src/bci_build/package/appcontainers.py

```diff
@@ -517,7 +517,11 @@ def _get_nginx_kwargs(os_version: OsVersion):
         license="GPL-2.0-only",
         package_list=[
             Package(name, pkg_type=PackageType.BOOTSTRAP)
-            for name in ("git-core",)
+            for name in (
+                "catatonit",
```

Comment:
The use case from https://github.com/rancher/gitjob/commit/b2a3eaf53efd is imho too niche for a general purpose git image, I hence would not include `catatonit` in the image.
```suggestion
```

Reply:
Not all containers need a separate process to be PID 1, actually so far it has been the exception rather than the rule.